DeepSpeech.ipynb

```diff
@@ -963,7 +963,7 @@
     "        # Execute operations of tower i on device i\n",
     "        with tf.device(available_devices[i]):\n",
     "            # Create a scope for all operations of tower i\n",
-    "            with tf.name_scope('tower_%d' % i) as scope:\n",
+    "            with tf.variable_scope('tower_%d' % i) as scope:\n",
```

Comment:
With this change, each tower's variables will also have a different scope, so the `tf.get_variable_scope().reuse_variables()` call below won't work for sharing the variables across the towers. We only want the ops to have a different scope for each tower, as we'll send them different input data, so that's why we (and the CIFAR-10 example) use `tf.name_scope()`.